Cargo.toml

```diff
@@ -22,6 +22,7 @@ members = [
   "tremor-pipeline",
   "tremor-script",
   "tremor-value",
+  "tremor-runtime-derive"
```

Comment:
Perhaps we can pick a better name for this? it's less about the runtimes than the connector traits.

Reply:
`tremor-runtime-connector-traits-derive`? Generally, the convention seems to be to keep proc macros in a crate with the same name as the main crate + `-derive`, but I agree this is a bit unspecific in this case.

Reply:
How about `tremor-connector-derive`. The convention makes sense if the derive is for what is in the trait (serde-derive, simd-json-derive etc), but that's not the case for us it's explicitly about connectors in our case, and down the road when the PDK comes in they'll probably be split out in an own crate.